httpgrpc/server/server.go

```diff
@@ -179,6 +179,19 @@ func HTTPRequest(r *http.Request) (*httpgrpc.HTTPRequest, error) {
 	}, nil
 }
 
+// UnwrapHTTPRequest unwraps an ordinary http.Request from an httpgrpc.HTTPRequest
+func UnwrapHTTPRequest(ctx context.Context, r *httpgrpc.HTTPRequest) (*http.Request, error) {
+	req, err := http.NewRequest(r.Method, r.Url, nopCloser{Buffer: bytes.NewBuffer(r.Body)})
+	if err != nil {
+		return nil, err
+	}
+	toHeader(r.Headers, req.Header)
+	req = req.WithContext(ctx)
+	req.RequestURI = r.Url
+	req.ContentLength = int64(len(r.Body))
```

Comment:
We don't need to repeat these steps in `Handle` above, do we?

Reply:
yep, just fixed